creator-node/scripts/start.sh

```diff
@@ -76,6 +76,7 @@ if [[ "$devMode" == "true" ]]; then
     if [ "$link_libs" = true ]
     then
         cd ../audius-libs
+        npm run dev &
```

Comment:
Do we need to run the dev server for cnode? It should be able to just import the ts directly right? (Though for consistency, using the bundled code is good)

Reply:
I think it can import ts directly, but if we make that assumption, then code may be written that works locally but doesn't work in prod since imports may be different. For example https://github.com/AudiusProject/audius-protocol/blob/07e2d6c9f3730d3e7e9100e121eebf9b366d363e/creator-node/src/snapbackSM/snapbackSM.js#L11 was a case that "just worked" for dev but not for prod (since you really shouldn't be referencing src code). Last thought here, which is that libs' "main" entrypoint points to `dist/index.js` so `import {Blah} from '@audius/libs'` wouldn't be hitting src anyway!

Reply:
yeah you're right! don't import directly